lib/commons/builder/legislative_term.rb

```diff
@@ -32,12 +32,11 @@ def output_relative
   end
 
   def ==(other)
-    other.instance_of?(self.class) &&
-      legislature == other.legislature &&
-      term_item_id == other.term_item_id &&
-      start_date == other.start_date &&
-      end_date == other.end_date &&
-      comment == other.comment
+    false unless other.instance_of? self.class
+    %i[legislature term_item_id start_date end_date comment position_item_id].each do |name|
```

Comment:
It might be neater to use `Enumerable.all?` here: https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-all-3F

Reply:
Yep; changed :-)